keyboards/lily58/keymaps/encoder/keymap.c

```diff
@@ -0,0 +1,231 @@
+#include QMK_KEYBOARD_H
+
+#ifdef PROTOCOL_LUFA
+  #include "lufa.h"
+  #include "split_util.h"
+#endif
+#ifdef SSD1306OLED
+  #include "ssd1306.h"
+#endif
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
```

Comment:
This shouldn't be needed anymore, as there are functions to check all of the rgb config settings, and to write to them.

Reply:
I never responded to this. I don't think I'll be making that change since it would require touching all the keymaps and changing how the value is read in lib/rgb_state_reader.c, and that is enough effort to warrant a separate PR.